### PR TITLE
Update Endpoint documentation

### PR DESCRIPTION
**Pull request by joshraker**
Update Endpoint documentation with more useful examples and descriptions.

### CODE REVIEW

**Comment by joshraker** [comment]
Making @turner-aptible reviewer because he raised this issue. cc: @madhuravius.

File: docs/resources/endpoint.md [hunk 1]
```diff
@@ -1,65 +1,111 @@
...
+  env_id        = data.aptible_environment.example.env_id
+  resource_id   = aptible_app.example.app_id
+  resource_type = "app"
+  process_type  = "cmd"
+  managed       = true
+  domain        = "www.example.com"
+}
+
+data "aws_route53_zone" "example" {
+  name = "example.com."
```
Comment (turner-aptible) [inline docs/resources/endpoint.md:39]: Don't need the `.` at the end "example.com"

**Review by turner-aptible — CHANGES_REQUESTED** [review]
Update the data block to remove the `.` at the end of `"example.com"`. More just formatting than anything. I think you "can" have it, but not required.

File: docs/resources/endpoint.md (hunk 1 above)
Comment (joshraker) [reply docs/resources/endpoint.md]: 👍 I just copied from the AWS example.

**Review by turner-aptible — APPROVED** [review]
Good to go

**Review by madhuravius — APPROVED** [review]

**Pull request merged**